### DISCUSSION TITLE
how to suppress logs from init/wait containers? just interested in main container logs

### DISCUSSION THREAD

**Discussion by tooptoop4** [argoproj/argo-workflows#10651] [Q&A]
how to suppress logs from init/wait containers? just interested in main container logs

**Comment by terrytangyuan** [comment]
In the UI, you can select the container.

**Reply by tooptoop4** [reply]
@terrytangyuan I need to reduce cost of logs being sent to log aggregation tool

**Reply by terrytangyuan** [reply]
There's currently no way to suppress that from Argo. You'd have to configure your log service.
